src/main/src/main/java/org/geoserver/catalog/SLDHandler.java

```diff
@@ -5,11 +5,28 @@
  */
```

Comment:
So general feedback that header needs to be updated (yeah I know it is not that important ...)